packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -296,11 +296,20 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       // Allow the caller to handle the key down
       onKeyDown?.(ev);
 
-      // Handle copy if focus is in the selected items list
+      // (I'm attempting to send a draft PR to see what it looks like if we violate lint,
+      // and get feedback on my first checkin).
+      // The explicit keycodes is probably the localization issue mentioned below
+      const isDel = ev.which === KeyCodes.del;
+      const isCut = (ev.shiftKey && isDel) || (ev.ctrlKey && ev.which === KeyCodes.x);
+      const isBackspace = !isDel && ev.which === KeyCodes.backspace && selectedItems.length > 0;
+      const needToCopy = isCut || (ev.ctrlKey && ev.which === KeyCodes.c);
+      const needToDelete = isBackspace || ((isCut || isDel) && focusedItemIndices.length > 0);
```

Comment:
I think it would be easier to read if this section had the check for ctrl-v, ctrl-c and backspace, then below the checks were if copy or cut, copy, if backspace or cut, delete

Reply:
Oh oops, didn't refresh. I think this version is ok too